content/projects/vgp/workflows/index.md

```diff
@@ -3,23 +3,23 @@
2.0" result="blur"/><feComponentTransfer in="blur" color-interpolation-filters="sRGB"><feFuncR type="linear" slope="0" intercept="0.0"/><feFuncG type="linear" slope="0" intercept="0.0"/><feFuncB type="linear" slope="0" intercept="0.0"/><feFuncA type="linear" slope="0.5" intercept="0"/></feComponentTransfer></filter></defs><g id="p.44"><path fill="#ffe599" d="m131.88321 287.25525l3.0 -12.0l34.51181 0l-3.0 12.0z" fill-rule="evenodd"/><path fill="#000000" d="m140.81444 283.37524l0 -3.8125l0.765625 0l0.90625 2.703125q0.125 0.375 0.171875 0.5625q0.078125 -0.203125 0.203125 -0.609375l0.921875 -2.65625l0.671875 0l0 3.8125l-0.484375 0l0 -3.1875l-1.109375 3.1875l-0.453125 0l-1.109375 -3.25l0 3.25l-0.484375 0zm6.2977448 -0.890625l0.46875 0.0625q-0.109375 0.421875 -0.421875 0.65625q-0.296875 0.234375 -0.78125 0.234375q-0.609375 0 -0.96875 -0.375q-0.359375 -0.375 -0.359375 -1.046875q0 -0.703125 0.359375 -1.078125q0.359375 -0.390625 0.9375 -0.390625q0.5625 0 0.90625 0.375q0.34375 0.375 0.34375 1.0625q0 0.046875 0 0.125l-2.0625 0q0.03125 0.453125 0.265625 0.703125q0.234375 0.234375 0.578125 0.234375q0.25 0 0.4375 -0.125q0.1875 -0.140625 0.296875 -0.4375zm-1.546875 -0.75l1.546875 0q-0.03125 -0.359375 -0.171875 -0.53125q-0.234375 -0.265625 -0.59375 -0.265625q-0.3125 0 -0.53125 0.21875q-0.21875 0.203125 -0.25 0.578125zm2.6038818 1.640625l0 -2.765625l0.421875 0l0 0.421875q0.15625 -0.296875 0.296875 -0.390625q0.140625 -0.09375 0.296875 -0.09375q0.25 0 0.484375 0.15625l-0.15625 0.4375q-0.171875 -0.109375 -0.34375 -0.109375q-0.15625 0 -0.28125 0.09375q-0.125 0.09375 -0.171875 0.25q-0.078125 0.25 -0.078125 0.546875l0 1.453125l-0.46875 0zm1.7586823 1.0625l-0.046875 -0.4375q0.15625 0.046875 0.265625 0.046875q0.15625 0 0.25 -0.0625q0.09375 -0.046875 0.15625 -0.140625q0.046875 -0.078125 0.140625 -0.34375q0.015625 -0.046875 0.046875 -0.125l-1.0625 -2.765625l0.515625 0l0.578125 1.609375q0.109375 0.296875 0.1875 0.640625q0.09375 -0.328125 0.203125 -0.640625l0.59375 -1.609375l0.46875 0l-1.0625 2.8125q-0.171875 0.453125 -0.265625 0.625q-0.125 0.234375 -0.28125 0.34375q-0.15625 0.109375 -0.390625 0.109375q-0.125 0 -0.296875 -0.0625zm2.6796875 -1.0625l0 -3.8125l0.46875 0l0 3.8125l-0.46875 0zm4.4609528 0l0 -0.34375q-0.265625 0.40625 -0.765625 0.40625q-0.328125 0 -0.609375 -0.171875q-0.28125 -0.1875 -0.4375 -0.515625q-0.140625 -0.328125 -0.140625 -0.75q0 -0.421875 0.125 -0.75q0.140625 -0.34375 0.421875 -0.515625q0.28125 -0.1875 0.609375 -0.1875q0.25 0 0.4375 0.109375q0.203125 0.109375 0.328125 0.265625l0 -1.359375l0.46875 0l0 3.8125l-0.4375 0zm-1.484375 -1.375q0 0.53125 0.21875 0.796875q0.234375 0.25 0.546875 0.25q0.296875 0 0.515625 -0.25q0.21875 -0.25 0.21875 -0.765625q0 -0.5625 -0.21875 -0.828125q-0.21875 -0.265625 -0.546875 -0.265625q-0.3125 0 -0.53125 0.265625q-0.203125 0.25 -0.203125 0.796875zm3.0882568 1.375l-0.4375 0l0 -3.8125l0.46875 0l0 1.359375q0.296875 -0.375 0.765625 -0.375q0.25 0 0.46875 0.109375q0.234375 0.09375 0.375 0.28125q0.15625 0.1875 0.234375 0.453125q0.09375
```

Comment:
comments for the image: 
- the workflow & QC text for WFs 7,8,9 are black when they're white in the rest of the boxes
- for the trio inputs, I think I might prefer "Illumina (mat.)" and "Illumina (pat.)" instead of the sex symbols, to just convey that it's read as from different individuals (ie the parents)
- for the decision point of "purging needed?" I suggest just having "Purging</br>?" so it matches the rest of the decision point
- for WF7, the outputs will be hap1 assembly and hap2 assembly, not primary&alternate
- for WF7, possibly just have the outputs as "purged contigs" to make it agnostic of the type of haplotype that went into it (since we can feed primary, altnernate, hap1, mat, etc)
- for the inputs, I would put HiC as a required input data, as that's the scaffolding data we do always try to generate

Reply:
oh gosh I didn't realize commenting on this line would put its entire length on the PR page, sorry